test/resources/PublicKey.spec.js

```diff
@@ -3,7 +3,7 @@
 var telnyx = require('../../testUtils').getTelnyxMock();
 var expect = require('chai').expect;
 
-describe('PublicKey Resource', function() {
+describe.skip('PublicKey Resource', function() {
```

Comment:
Is this resource not available anymore? Could you also confirm this one?

Reply:
yea, in service of python too 

https://github.com/team-telnyx/telnyx-python/blob/master/tests/api_resources/test_public_key.py
look on line 9